+++ hyperram.c
@@ -11,26 +11,26 @@
 
 static void hyperram_write_reg(uint16_t reg_addr, uint16_t data) {
     /* Write data to the register */
-    hyperram_reg_wdata_write(data);
-    hyperram_reg_control_write(
-        1        << CSR_HYPERRAM_REG_CONTROL_WRITE_OFFSET |
-        0        << CSR_HYPERRAM_REG_CONTROL_READ_OFFSET  |
-        reg_addr << CSR_HYPERRAM_REG_CONTROL_ADDR_OFFSET
+    hyperram_hyperram_reg_wdata_write(data);
+    hyperram_hyperram_reg_control_write(
+        1        << CSR_HYPERRAM_HYPERRAM_REG_CONTROL_WRITE_OFFSET |
+        0        << CSR_HYPERRAM_HYPERRAM_REG_CONTROL_READ_OFFSET  |
+        reg_addr << CSR_HYPERRAM_HYPERRAM_REG_CONTROL_ADDR_OFFSET
     );
     /* Wait for write to complete */
-    while ((hyperram_reg_status_read() & (1 << CSR_HYPERRAM_REG_STATUS_DONE_OFFSET)) == 0);
+    while ((hyperram_hyperram_reg_status_read() & (1 << CSR_HYPERRAM_HYPERRAM_REG_STATUS_DONE_OFFSET)) == 0);
  }
 
 static uint16_t hyperram_read_reg(uint16_t reg_addr) {
     /* Read data from the register */
-    hyperram_reg_control_write(
-        0        << CSR_HYPERRAM_REG_CONTROL_WRITE_OFFSET |
-        1        << CSR_HYPERRAM_REG_CONTROL_READ_OFFSET  |
-        reg_addr << CSR_HYPERRAM_REG_CONTROL_ADDR_OFFSET
+    hyperram_hyperram_reg_control_write(
+        0        << CSR_HYPERRAM_HYPERRAM_REG_CONTROL_WRITE_OFFSET |
+        1        << CSR_HYPERRAM_HYPERRAM_REG_CONTROL_READ_OFFSET  |
+        reg_addr << CSR_HYPERRAM_HYPERRAM_REG_CONTROL_ADDR_OFFSET
     );
     /* Wait for read to complete */
-    while ((hyperram_reg_status_read() & (1 << CSR_HYPERRAM_REG_STATUS_DONE_OFFSET)) == 0);
-    return hyperram_reg_rdata_read();
+    while ((hyperram_hyperram_reg_status_read() & (1 << CSR_HYPERRAM_HYPERRAM_REG_STATUS_DONE_OFFSET)) == 0);
+    return hyperram_hyperram_reg_rdata_read();
 }
 
 /* Configuration and Utility Functions */
@@ -91,7 +91,7 @@
     printf("HyperRAM init...\n");
 
     /* Compute Latency settings */
-    core_clk_ratio  = (hyperram_status_read() >> CSR_HYPERRAM_STATUS_CLK_RATIO_OFFSET) & 0xf;
+    core_clk_ratio  = (hyperram_hyperram_status_read() >> CSR_HYPERRAM_HYPERRAM_STATUS_CLK_RATIO_OFFSET) & 0xf;
     printf("HyperRAM Clk Ratio %d:1\n", core_clk_ratio);
     core_latency_setting = hyperram_get_core_latency_setting(CONFIG_CLOCK_FREQUENCY / core_clk_ratio);
     chip_latency_setting = hyperram_get_chip_latency_setting(CONFIG_CLOCK_FREQUENCY / core_clk_ratio);
@@ -109,9 +109,9 @@
 #endif
 
     /* Configure Latency on HyperRAM Core */
-    core_latency_mode = (hyperram_status_read() >> CSR_HYPERRAM_STATUS_LATENCY_MODE_OFFSET) & 0b1;
+    core_latency_mode = (hyperram_hyperram_status_read() >> CSR_HYPERRAM_HYPERRAM_STATUS_LATENCY_MODE_OFFSET) & 0b1;
     printf("HyperRAM %s Latency: %d CK (X1)\n", (core_latency_mode == 0) ? "Fixed" : "Variable", core_latency_setting);
-    hyperram_config_write(core_latency_setting << CSR_HYPERRAM_CONFIG_LATENCY_OFFSET);
+    hyperram_hyperram_config_write(core_latency_setting << CSR_HYPERRAM_HYPERRAM_CONFIG_LATENCY_OFFSET);
     printf("HyperRAM Drive Strength: %d ohm\n", drive_strength_ohms);
 
     /* Configure HyperRAM Chip */
@@ -138,7 +138,7 @@
         (HYPERRAM_CONFIG_0_REG_DPD_DISABLED       << HYPERRAM_CONFIG_0_REG_DPD_OFFSET)
     );
     /* Enable Variable Latency on HyperRAM Chip */
-    if (hyperram_status_read() & 0x1) {
+    if (hyperram_hyperram_status_read() & 0x1) {
         config_reg_0 &= ~(1                                  << HYPERRAM_CONFIG_0_REG_FLE_OFFSET);
         config_reg_0 |=  (HYPERRAM_CONFIG_0_REG_FLE_DISABLED << HYPERRAM_CONFIG_0_REG_FLE_OFFSET);
     }
 }
\ No newline at end of file
